### DISCUSSION TITLE
Xpose not reading NLMIX fit object

### DISCUSSION THREAD

**Discussion by Ravua1992** [nlmixr2/nlmixr2#266] [General]
I am getting the following error when trying to read model after fitting. 

works fine.:
```
plot(fit)
```

Below code does not: 
```
xpose_data_nlmixr(fit)
Error in `[.data.frame`(d, , i) : undefined columns selected
```

**Comment by billdenney** [comment]
@Ravua1992 , can you please share a reproducible example?
